terraform/environments/digital-prison-reporting/data_ingestion_pipeline.tf

```diff
@@ -0,0 +1,197 @@
+# Lambda which notifies step function when DMS task stops
+module "step_function_notification_lambda" {
```

Comment:
Since step function has no way of know when the DMS task has finished, this Lambda is used to notify the step function.